src/types/index.ts

```diff
@@ -456,7 +456,6 @@ export interface IGAMetaTx {
 export interface ITx {
   abiVersion?: number;
   accountId?: AccountAddress;
-  aexnType?: 'aex9';
```

Comment:
`aexnType` word was not used anywhere in the repository besides this line.

Reply:
This is where is came from https://github.com/aeternity/ae_mdw/pull/1591.